src/main/java/no/difi/statistics/elasticsearch/ElasticsearchQueryService.java

```diff
@@ -1,17 +1,15 @@
 package no.difi.statistics.elasticsearch;
 
-import no.difi.statistics.model.MeasurementDistance;
-import no.difi.statistics.model.TimeSeriesDefinition;
-import no.difi.statistics.model.TimeSeriesPoint;
-import no.difi.statistics.model.PercentileFilter;
-import no.difi.statistics.model.QueryFilter;
 import no.difi.statistics.QueryService;
+import no.difi.statistics.model.*;
```

Comment:
Det er ikke et dramatisk problem, men IDEA burde konfigureres til å ikke bruke *-import (som er litt dårlig praksis, siden man ikke vet akkurat hva som importeres. Fikses i IDEA ved å sette en veldig høy terskel før imports fra en package omskrives til * fra den pakken)

Reply:
Ja, enig. Har ændret (tilbage) så den henter hver enkelt pakke.

Namespacet model importeres også via client-lib og common-api-lib i tillæg til at det er i selve projektet. Hvad er praksis der?

Reply:
Model / service / config er en utbredt måte å organisere klasser i bl. a. Spring-applikasjoner. Navnerommet (foran .model) er derfor nødvendig for å skille mellom dem.
 Igjen har @digdirerik argumentert for en annen organisering, men det var gjerne i sammenheng med et klient-bibliotek.